### PR TITLE
Add start-up test for flutter_view sample

### PR DESCRIPTION
**Pull request by szakarias**
We settle for a start-up test for now. Since this app has a native view we cannot use the Flutter driver to test.

### CODE REVIEW

File: dev/devicelab/lib/tasks/perf_tests.dart [hunk 1]
```diff
@@ -67,6 +67,12 @@ TaskFunction createGalleryBackButtonMemoryTest() {
   );
 }
 
+TaskFunction createFlutterViewStartupTest() {
+  return new StartupTest(
```
Comment (yjbanov) [inline dev/devicelab/lib/tasks/perf_tests.dart]: The `StartupTest` will publish start-up metrics to [the dashboard](https://flutter-dashboard.appspot.com/benchmarks.html). I'm not sure these metrics will be useful? Can we add a `reportMetrics` to `StartupTest`'s constructor, default it to `true` and set it to `false` for this test?

Comment (szakarias) [inline dev/devicelab/lib/tasks/perf_tests.dart]: No, I don't see the use of them either in this case. :-) Added the flag.

File: dev/devicelab/manifest.yaml [hunk 2]
```diff
@@ -144,6 +144,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["has-android-device"]
 
+  flutter_view__start_up:
+    description: >
+      Measures the startup time of the Flutter View app on Android.
```
Comment (yjbanov) [inline dev/devicelab/manifest.yaml]: Let's reword this to "Verifies that Flutter View can be used from an Android project."

File: dev/devicelab/manifest.yaml [hunk 3]
```diff
@@ -190,6 +196,12 @@ tasks:
     stage: devicelab_ios
     required_agent_capabilities: ["has-ios-device"]
 
+  flutter_view_ios__start_up:
+    description: >
+      Measures the startup time of the Flutter View app on iOS.
```
Comment (yjbanov) [inline dev/devicelab/manifest.yaml]: Similar rewording here.

**Review by yjbanov — APPROVED** [review]
LGTM
